src/evergreen/task_annotations.py

```diff
@@ -0,0 +1,81 @@
+"""Models to working with task annotations."""
+from typing import TYPE_CHECKING, Any, Dict, List
+
+from evergreen.base import _BaseEvergreenObject, evg_attrib, evg_datetime_attrib
+
+if TYPE_CHECKING:
+    from evergreen.api import EvergreenApi
+
+
+class Source(_BaseEvergreenObject):
```

Comment:
I don't think you should ever have to use this. We add that to all issue links and notes. (though I'm guessing this is autogenerated)

Reply:
This code is just for reading the data we get back from the API, it isn't used for writing anything.